src/loki/values/values.yaml

```diff
@@ -5,11 +5,28 @@ fullnameOverride: loki
 # -- Overrides the chart's cluster label
 clusterLabelOverride: null
 
+# Prevent startup issues with service existence/resolution
+memberlist:
+  service:
+    publishNotReadyAddresses: true
+
 loki:
   storage:
-    type: "filesystem"
+    bucketNames:
+      chunks: uds
+      ruler: uds
+      admin: uds
+    type: s3
+    s3:
+      endpoint: http://minio.uds-dev-stack.svc.cluster.local:9000
```

Comment:
So...I can't think of a good way to handle this...but I believe for IRSA you have to leave endpoint empty? Which means someone using IRSA will have to wipe out this value to get it to work. Can't think of a way to leave this empty by default but make CI happy...so maybe we just document the IRSA setup? Doesn't have to be done in this PR unless there is an obvious fix.

Reply:
Yea, I think you're right.  I believe `s3forcepathstyle` would have to be unset (or false) as well.  Happy to document in a follow on PR or maybe as an addition to the EKS nightly testing.

Reply:
Yeah that works - we'll probably have a better grasp with the EKS testing and can point to an example.